src/sentry/static/sentry/app/components/acl/feature.tsx

```diff
@@ -66,15 +67,13 @@ class Feature extends React.Component<FeatureProps> {
     /**
      * Specify the key to use for hookstore functionality.
      *
-     * The hookstore key that will be checked is:
-     *
-     *     feature-disabled:{hookName}
+     * The hookName should be prefixed with `feature-disabled`.
      *
      * When specified, the hookstore will be checked if the feature is
      * disabled, and the first available hook will be used as the render
      * function.
      */
-    hookName: PropTypes.string,
+    hookName: PropTypes.string as any,
```

Comment:
Because the type of `PropTypes.string` isn't compatible with the `HookName` type. I would have to make this a `PropTypes.oneOf()` type thing for it to be compatible.